examples/website/i3s/app.js

```diff
@@ -121,6 +123,50 @@ export default class App extends PureComponent {
     this._onSelectTileset(tileset);
   }
 
+  /**
+   * Get elevation data for TerrainLayer
+   * If MapboxAccessToken is rovided we will use mapbox data.
+   * If not we will use Mapzen Terrain Tile and ArcGIS texture.
+   * Docs - https://github.com/tilezen/joerd/tree/master/docs
+   */
+  getTerrainLayerData() {
+    const MAPBOX_TOKEN = process.env.MapboxAccessToken;
```

Comment:
I think it would be clearer here if you split this body into two functions:

```js
getTerrainLayerData() {
  const MAPBOX_TOKEN = process.env.MapboxAccessToken;
  if (MAPBOX_TOKEN) {
    return getMapboxTerrainLayerData(MAPBOX_TOKEN);
  } else {
    return getOpenTerrainLayerData();
  }
}
```
Then it's clear you're only using the `MAPBOX*` variables if the `MapboxAccessToken` env actually exists.